Update style.css

elic4vet opened the pull request:
I added some styles on the hero section, i will continue working on this subject later. Hope it's okay for you :)

elic4vet commented:
I had to repost my pull request because i couldn't find my changes concerning the html file.  Hope it's okay for you :)

elic4vet commented:
i created the about us section and added the corresponding link that navigates from home to the about section and back to home

basiacarvalho commented on css/style.css:
```diff
@@ -190,6 +190,35 @@ ul {
+  font-family: "Roboto-Black", sans-serif;
+  font-size: 2.25rem;
+  margin-bottom: 10px;
+}
```
Decide whether you want to always leave one line space between style declarations or have it all one by one. I read on the internet that is better to keep one empty line in style for readability (so I always separate style declarations with one line).

basiacarvalho commented on index.html:
```diff
@@ -24,10 +24,9 @@
     <main>
       <!--1st section - Hero -->
-      <section class="section">
+      <section class="hero">
```
You can leave the common class "section" in the section tag, and apart from that add the second class name "hero" and put in "hero" only what varies from the normal "section" styling.

basiacarvalho reviewed (approved):
Hi, I left 2 comments :)

elic4vet commented:
I added the course link on the about us section

elic4vet commented:
•	I added an image to the hero section.
•	I also added an image container to better divide and organize these two sections.
•	I added styles for the two divs and the image.

- [ ] 	I need to work later on the style of the page and @media queries for the mob + tablet version.

basiacarvalho commented on index.html:
```diff
@@ -16,24 +16,35 @@
+            <img 
+              class="hero-img" 
+              src="./images/heroimg.jpg" 
+              alt="a person sitting in front of a computer"
```
I would probably write alt="an open laptop with code display on a desk with a plant and a mug" for the hero img

basiacarvalho reviewed (changes requested):
Hi, 

One thing to fix - look at my proposition of an "alt" for hero img :) 
"an open laptop with code displayed on a desk with a plant and a mug"

elic4vet commented:
I changed the text description of the alt as suggested before
